### PR TITLE
Reorganizing the VB TOC

### PR DESCRIPTION
**Pull request by stevehoag**
Updated the Programming in Visual Basic secton

### CODE REVIEW

**Comment by BillWagner** [comment]
@stevehoag I like the way you did this. I tried a different route with C# in PR #1318  We should try to measure which customers like better (or if it's a mix at different times.)

I do see quite a few warnings in the build report. It looks like a macro or somethings added 'visual-basic/programming' into the destination links a few times. Once that gets fixed, I'll merge.

**Comment by stevehoag** [comment]
@BillWagner This is just the first set of TOC changes - I think we can way and merge them all at once.

I can't figure out what's causing the warnings - the 81 warnings correspond with the 81 entries that I removed from the TOC, and as you noted, they have  an extra 'visual-basic/programming' insert into the original paths.

File: docs/visual-basic/developing-apps/programming/toc.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+# [Accessing Computer Resources](visual-basic/developing-apps/programming/computer-resources/index.md)
```
Comment (BillWagner) [inline docs/visual-basic/developing-apps/programming/toc.md]: I think this file has the 81 warnings.  The target starts with a relative path, and when you moved these entries to a new TOC, the location moved.  I'd try to change this:

``` 
# [Accessing Computer Resources](visual-basic/developing-apps/programming/computer-resources/index.md) 
```

to this:

``` 
# [Accessing Computer Resources](computer-resources/index.md) 
```

And make similar changes throughout this file.

**Comment by BillWagner** [comment]
@stevehoag I think its in the new TOC you added in the `visual-basic/developing-apps/programming` directory. I put a note on the first line there.

**Comment by stevehoag** [comment]
@BillWagner Thanks Bill - that did indeed fix it. I was thrown off by the build report showing that the warnings were in docs\toc.md.

**Comment by stevehoag** [comment]
Closing and reopening to force build

**Comment by stevehoag** [comment]
This should be ready to merge now.

**Comment by mairaw** [comment]
I'm still seeing TOC levels like that (I just started at a random location). Is that expected?
https://review.docs.microsoft.com/en-us/dotnet/articles/visual-basic/programming-guide/concepts/linq/xattribute-class-overview?branch=pr-en-us-1348

image

Whar are exactly the TOC changes I should be looking at?

**Comment by mairaw** [comment]
and why are we changing the file names from index.md to foldername.md?

**Comment by stevehoag** [comment]
Closing and reopening to force build.

**Comment by BillWagner** [comment]
I looked through the changes, and visited quite a few of the pages on the review branch.  This LGTM.  Let's :shipit: 🎆

**Pull request merged**